### ISSUE TITLE
Placeholder is not announced by screen readers

### ISSUE THREAD

**Issue by quentinderoubaix** [ng-select/ng-select#2498]
### Describe the bug

Screen readers fetch and read placeholders associated to inputs.
ng-select behaves / displays like a standard input but the placeholder is not part of the `input` itself.

Proposal:
add an `aria-placeholder` in the input with the content of the ng-select input placeholder

Note that this raises the question of what to do when a placeholder template is used...

### Steps to Reproduce

Use NVDA, focus a ng-select that was setup with a placeholder, and notice how the placeholder is not announced.

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://ng-select.github.io/ng-select/#/data-sources

### Expected behaviour

placeholder should be announced by screen readers

### What version of the library you see this issue?

14.2.0

### What version of Angular are you using?

Angular 19
